pkg/maxscale/config/defaults.go

```diff
@@ -0,0 +1,38 @@
+package config
+
+import (
+	"fmt"
+
+	mariadbv1alpha1 "github.com/mariadb-operator/mariadb-operator/api/v1alpha1"
+)
+
+const (
+	queryClassifierCacheLimitFraction = 0.15
```

Comment:
It might be worth adding a comment with a link to the MaxScale documentation to understand where this factor comes from:
https://mariadb.com/kb/en/mariadb-maxscale-6-mariadb-maxscale-configuration-guide/#query_classifier_cache_size

Reply:
Added